patchwork-gui/src/main/java/net/patchworkmc/mixin/gui/MixinAbstractButtonWidget.java

```diff
@@ -30,11 +30,17 @@
 	@Shadow
 	protected int height;
 
+	protected int packedFGColor = 0;
```

Comment:
Is this correct or is a `@Shadow` missing here? The field does not seem to do anything, correct me if I'm wrong.

Reply:
`@Shadow` would not work here as `packedFGColor` is not a field in `AbstractButtonWidget`.